### PR TITLE
Fixed the overlapping of subheadings

### LINKED ISSUE #3332: Subheadings overlap in left menu
**Actual Behaviour**
When scrolled down, the subheadings in the left menu overlap over each other

**Expected Behaviour**
The subheadings should not overlap over each other

**Steps to reproduce it**
- Go to the Home page and change the screen resolution to mobile.
- Open the left menu and scroll down to see the error.

**Screenshots of the issue**
Screenshot (47)

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Dishebh**
Fixes #3332 

Changes: Fixed the overlapping issue of subheadings in the left menu on scroll.

Demo Link: https://pr-3333-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by Dishebh** [comment]
@akshatnitd pls review. Thanks

**Comment by Dishebh** [comment]
@akshatnitd pls merge. Thanks

File: src/components/About/Overview/index.js [hunk 1]
```diff
@@ -656,6 +656,28 @@ class Overview extends Component {
               Don&apos;t forget, these are only a few 😊
             </DescriptionText>
             <RowDiv>
+<<<<<<< HEAD
```
Comment (samagragupta) [inline src/components/About/Overview/index.js]: Remove the unwanted code.

File: src/components/cms/BrowseSkill/skillLoader.js [hunk 2]
```diff
@@ -9,8 +9,13 @@ const Container = styled.div`
 `;
 
 const ContentContainer = styled.div`
+<<<<<<< HEAD
```
Comment (samagragupta) [inline src/components/cms/BrowseSkill/skillLoader.js]: Here also

**Comment by Dishebh** [comment]
Ahh! Looks like my branch got messed up, accidentally. I will close the PR and open a fresh one with the same commits. Sorry!

**Pull request closed without merging**